### ISSUE TITLE
1

### ISSUE THREAD

**Issue by UnlcleTian** [UnlcleTian/kaggle#1]
import numpy as np
import pandas as pd
# from sklearn.model_selection import KFold
from lightgbm import LGBMClassifier

import matplotlib.pyplot as plt
import seaborn as sns
import gc

# data_train = pd.read_csv(r'C:\Users\guanhua.wang\Downloads\data\train.csv')

# data_test = pd.read_csv(r'C:\Users\guanhua.wang\Downloads\data\test.csv')
# data_his_tra = pd.read_csv(r'C:\Users\guanhua.wang\Downloads\data\historical_transactions.csv')

# data_mer = pd.read_csv(r'C:\Users\guanhua.wang\Downloads\data\merchants.csv')

data_new_mer = pd.read_csv(r'C:\Users\guanhua.wang\Downloads\data\new_merchant_transactions.csv')

print('new shape:',data_new_mer.shape)
print('type:',data_new_mer.dtypes)

# count the null cell??
# print('null:',data_new_mer.isnull())

# A:2,B:3,?

data_new_mer_dum1 = pd.get_dummies(data_new_mer.category_1,prefix='cate1',dummy_na=True)
data_new_mer_dum2 = pd.get_dummies(data_new_mer.category_3,prefix='cate3',dummy_na=True)

print(data_new_mer_dum1.head())
print(data_new_mer_dum1.shape)

plot picture for discribution of data column

city_id,merchant_category_id
state id vs score

data_new_mer = pd.concat([data_new_mer,data_new_mer_dum1,data_new_mer_dum2],axis=1 ).drop(['category_1','category_3'],axis=1)

print(data_new_mer.head())
